NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -699,6 +701,22 @@ public StateQuery()
                     return result;
                 }
             );
+
+            Field<StringGraphType>(
+                name: "cachedSheet",
+                arguments: new QueryArguments(
+                    new QueryArgument<StringGraphType>
+                    {
+                        Name = "tableName"
+                    }
+                ),
+                resolve: context =>
+                {
+                    var tableName = context.GetArgument<string>("tableName");
+                    var cacheKey = Addresses.GetSheetAddress(tableName).ToString();
+                    return context.Source.StateMemoryCache.SheetCache.GetSheet(cacheKey);
+                }
+            );
```

Comment:
Where is this API used? For debugging?

If then, though it may not be a recommended way, how about making it as a `@deprecated` query to prevent use by others?

Reply:
It was created to be used to compare chain and cached sheet data when operating a node.